ansible/roles/ocp4-workload-cnvlab/tasks/workload.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# To Do: Implement your workload deployment tasks here
+# -------------------------------------------------------------------------
+
+- name: Create project for workbook
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'project.yaml') }}"
+
+- name: Clone OpenShift virt repositories
+  git:
+    repo: "https://github.com/RHFieldProductManagement/openshift-virt-labs.git"
+    dest: "~/openshift-virt-labs"
+    version: "rhpds"
+
+- name: Deploy Workbook Instance for CNV Labs
+  block:
+  - name: Set project to workbook
+    shell: oc project workbook
+
+  - name: Configure and apply build template
+    shell: oc process -f ~/openshift-virt-labs/docs/build-template.yaml -p NAME=cnv | oc apply -f -
+
+  - name: Start the build
+    shell: oc start-build cnv --from-dir=openshift-virt-labs/docs/
+
+  - name: Configure and deploy the deploy template
+    shell: oc process -f ~/openshift-virt-labs/docs/deploy-template.yaml -p NAME=cnv -p IMAGE_STREAM_NAME=cnv | oc apply -f -
+
+- name: Give cluster-admin privileges to CNV user
+  command: oc adm policy add-cluster-role-to-user cluster-admin -z cnv
```

Comment:
This could also be done with `k8s` and a YAML file representing the ClusterRoleBinding. Example: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/templates/cluster_role_binding.j2

Reply:
Done, thx!